nerdlets/signal-selection/index.js

```diff
@@ -219,15 +219,17 @@ const SignalSelectionNerdlet = () => {
           levelId,
           stepId: step?.id,
           signals: [
-            ...(selectedEntities || []).map(({ guid, name }) => ({
+            ...(selectedEntities || []).map(({ guid, name, included }) => ({
               guid,
               name,
               type: SIGNAL_TYPES.ENTITY,
+              included: included !== undefined ? included : true,
```

Comment:
Not sure what the exact logic here, but if you're just checking to see is `included` is truthy, it's simpler to do this...
```javascript
included: included || true,
```

Reply:
I originally had that and it didnt work - I'm checking if `included` exists in the object.